src/components/touchableOpacity/index.tsx

```diff
@@ -112,10 +116,10 @@ class TouchableOpacity extends PureComponent<Props, {active: boolean}> {
   }
 
   render() {
-    const {useNative, style, modifiers, forwardedRef, ...others} = this.props;
+    const {useNative, activeScale, style, modifiers, forwardedRef, ...others} = this.props;
     const {borderRadius, paddings, margins, alignments, flexStyle} = modifiers;
 
-    if (useNative) {
+    if (useNative || !!activeScale) {
```

Comment:
It's a rare case but it won't work if the `activeScale` is equal to `0` even though it's a valid value.

Reply:
Fixed
Though passing `0` to activeScale is really weird (: